extensions/amp-wordpress-embed/amp-wordpress-embed.md

```diff
@@ -0,0 +1,62 @@
+---
+$category@: presentation
+formats:
+  - websites
+teaser:
+  text: Embeds a WordPress post.
+---
+
+<!--
```

Comment:
I think this needs to be removed.

Reply:
@pradeep910 These tags are using in all readme.md files for the extensions, for example in the [soundcloud](https://raw.githubusercontent.com/rtCamp/amphtml/main/extensions/amp-soundcloud/amp-soundcloud.md) extension we have 
```
---
$category@: media
formats:
  - websites
teaser:
  text: Displays a Soundcloud clip.
---
```

Reply:
Yes, I think we should include it ([YAML metadata](https://github.blog/2013-09-27-viewing-yaml-metadata-in-your-documents/)). All of the extension have YAML metadata.

GitHub renders it as:
<img width="961" alt="image" src="https://user-images.githubusercontent.com/75766877/122745874-147c6480-d2a7-11eb-98d3-ed47e8bdd3f8.png">

Reply:
I was referring to this comment - https://github.com/rtCamp/amphtml/pull/2/files#diff-253733eb9b30de47df4e2c93a894f252ae66adaefc9459ca54459ca862344099R10-R16

Reply:
Sorry for misunderstanding.

This can be removed - https://github.com/rtCamp/amphtml/pull/2/files#diff-253733eb9b30de47df4e2c93a894f252ae66adaefc9459ca54459ca862344099R10-R16

Reply:
@pradeep910 Ohh, yes, those lines should be removed. I'm going to push a new commit for this. Thanks.

Reply:
Comment removed.